arabic/unpacked/html.js

```diff
@@ -0,0 +1,90 @@
+MathJax.Hub.Config({
+  'HTML-CSS': {
+    undefinedFamily: 'Amiri',
+    styles: {
+      '.mfliph': {
+        'display': 'inline-block !important',
+        '-moz-transform': 'scaleX(-1)',
+        '-webkit-transform': 'scaleX(-1)',
+        '-o-transform': 'scaleX(-1)',
+        'transform': 'scaleX(-1)',
+        '-ms-filter': 'fliph',
+        'filter': 'fliph'
+      },
+      '.mar': {
+        'font-family': 'Amiri !important',
+        'font-style': 'normal !important'
+      }
+    }
+  }
+});
+
+
+MathJax.Hub.Register.StartupHook('mml Jax Ready', function () {
+  MathJax.Hub.Register.StartupHook('HTML-CSS Jax Ready', function () {
+    MathJax.Hub.Register.StartupHook('Arabic TeX Ready', function () {
+      var MML = MathJax.ElementJax.mml;
+
+      var flipHorizontalElement = function (token, element) {
+        var className = '';
+
+        if ('ar' === token.Get('fontLang')) {
+          className += ' ' + 'mar';
+        }
+
+        if (token.Get('fliph')) {
+          var flipElement = document.createElement('span');
+          className += ' ' + 'mfliph';
+
+          flipElement.className += ' ' + className;
```

Comment:
Since the `flipElement` will have an empty `className`, I don't think you need to add another space here (you already have one at the beginning of the `className` variable anyway).